graphrag/query/structured_search/drift_search/action.py

```diff
@@ -39,7 +39,11 @@ def __init__(
         self.follow_ups: list[DriftAction] = (
             follow_ups if follow_ups is not None else []
         )
-        self.metadata: dict[str, Any] = {}
+        self.metadata: dict[str, Any] = {
```

Comment:
I would like to maintain the existing `self.metadata` to record token usage. But if the same `DriftAction` object is being used for different queries, then we will need to reset these counters per query.